src/dataLayer/mongo/user-dataLayer.test.js

```diff
@@ -108,3 +109,26 @@ describe('getUser', () => {
     });
   });
 });
+
+describe('deleteUser', () => {
+  it('should delete an existing user', async done => {
+    const result = await createUser({}, {}, validContext);
+    const { accountLinkId } = result;
+    const response = await deleteUser({}, { accountLinkId }, validContext);
+    expect(response.statusCode).toEqual(204);
+    expect(response.message).toBeTruthy();
```

Comment:
Try to use snapshots instead, see the other tests and https://facebook.github.io/jest/docs/en/snapshot-testing.html for reference.